src/service/TrustMapperToPageService.ts

```diff
@@ -0,0 +1,66 @@
+import { TrustHistoricalBeneficialOwner } from '@companieshouse/api-sdk-node/dist/services/overseas-entities/types';
```

Comment:
These are not services, create an util for the trust work on the utils directory and rename the file please.

Use the model defined locally, consistency please